unity/Assets/Scripts/Logic/TileStack.cs

```diff
@@ -6,27 +6,20 @@
 using System.Linq;
 using UnityEngine;
 
-public class TileStack 
+public static class TileStack 
```

Comment:
Instead of making the whole class static, can we try something along these lines:

[GetComponent](https://stackoverflow.com/a/37399263/6163381)

Since the unity scripts for stack and map are both part of the same object, we should be able to access them through GetComponent. Then we wouldn't need a static TileStack and we could keep the implementation prior to this commit.